src/WaterPokemon.java

```diff
@@ -0,0 +1,40 @@
+public class WaterPokemon extends Pokemon{
+    private int depth;
+    private int shellthickness;
+
+    public WaterPokemon(String name, int depth, int shellthickness) {
+        super(name, "Water");
```

Comment:
Het is geen good practise om dit soort "magic Strings" te definieren, zoals "Water" hier. Deze string komt uit het niets en je kunt ook niet direct zien wat het betekend. 
Je kunt dit beter opslaan in een contante `private String POKEMONTYPE = "Water"`
Als je dat op deze manier doet, zie je meteen dat je het POKEMONTYPE meegeeft aan de super constructor en dat de waarde daarvan "Water" is.